packages/teleterm/build_resources/linux/after-install.tpl

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -eu
+
+###
+# Default after-install.tpl copied from electron-builder.
+# https://github.com/electron-userland/electron-builder/blob/v24.0.0-alpha.5/packages/app-builder-lib/templates/linux/after-install.tpl
+###
+
+# SUID chrome-sandbox for Electron 5+
+chmod 4755 "/opt/${sanitizedProductName}/chrome-sandbox" || true
+
+update-mime-database /usr/share/mime || true
+update-desktop-database /usr/share/applications || true
+
+###
+# Custom after-install.tpl script.
+###
+
+APP="/opt/${sanitizedProductName}"
+BIN=/usr/local/bin
+TSH_SYMLINK_SOURCE=$APP/resources/bin/tsh
+TSH_SYMLINK_TARGET=$BIN/tsh
+
+# Create $BIN if it doesn't exist.
+[ ! -d "$BIN" ] && mkdir -p "$BIN"
+
+# Link to the Electron app binary.
+ln -sf "$APP/${executable}" "$BIN/${executable}"
+
+# Link to the bundled tsh if the symlink doesn't exist already. Otherwise echo a message unless the
+# link points to teleport-connect's tsh already.
+# Does TSH_SYMLINK_TARGET not exist?
+if [ ! -e "$TSH_SYMLINK_TARGET" ]; then
+  ln -s "$TSH_SYMLINK_SOURCE" "$TSH_SYMLINK_TARGET"
+else
+  message="${executable}: Skipping symlinking $TSH_SYMLINK_TARGET to $TSH_SYMLINK_SOURCE"
+
+  # Is TSH_SYMLINK_TARGET a symlink?
+  if [ -L "$TSH_SYMLINK_TARGET" ]; then
+    # Does TSH_SYMLINK_TARGET point at something else than TSH_SYMLINK_SOURCE?
+    # If TSH_SYMLINK_TARGET exists and it points at TSH_SYMLINK_SOURCE already, don't do anything.
+    if [ ! "$TSH_SYMLINK_TARGET" -ef "$TSH_SYMLINK_SOURCE" ]; then
+      message+=" because the symlink already exists and it points to $(realpath $TSH_SYMLINK_TARGET)."
+      echo "$message"
+    fi
+  else
+    message+=" because $TSH_SYMLINK_TARGET already exists and it isn't a symlink."
+    echo "$message"
+  fi
+fi
+
+# DELETE IN 13.0.0
```

Comment:
Hmm, wy in v13?

Reply:
Ah, I figured that there might be people who would upgrade from, let's say, 11.0.1 straight to 12.x. If we removed this in 12.0, they'd be left with a broken symlink on their system. So hopefully if we do this in v13 there will be less people with that problem.

But! When I was writing this comment I realized that the removal of the old symlink is completely unnecessary. When people upgrade from an old version, their package manager is still going to run after-remove from the old version which is going to remove the symlink.

https://github.com/electron-userland/electron-builder/blob/v24.0.0-alpha.5/packages/app-builder-lib/templates/linux/after-remove.tpl